### DISCUSSION TITLE
Controlnet with Openvino SD Webui

### DISCUSSION THREAD

**Discussion by JGrant15** [openvinotoolkit/stable-diffusion-webui#56] [Q&A]
Hi there, I have managed to get the A1111 Webui running using the Arc A770 and the OpenVino acceleration script, one issue I am running into is the controlnet extension is completely ignored when I use the custom script, if I leave the script disabled controlnet is used but obviously image generation times are much slower, once I re-enable the script control net is completely ignored even though I can see the generated textures (canny image for example). Any help here would be greatly appreciated as  can see in the script controlnet is referenced but I can't seem to find anything related to my issue online.

**Comment by JGrant15** [comment]
I have managed to get Controlnet working by using the Control_net branch but it has a few bugs, I have opened an issue on the problems I've been experiencing so far

**Reply by jajjjo** [reply]
Experiencing exactly the same issue on intel UHD. Could you please elaborate on how you managed to get it to work? Just change repository branch?

**Reply by silvertuanzi** [reply]
In fact, controlnet branch has already been merged into master.
It should be supported now, but I suffered the same problem with Intel UHD.
The controlnet info is even written into PNG info, but the output image is obviously not related to controlnet.

**Reply by tuanakotta** [reply]
what do you mean by control_net branch?
